src/views/EditNote/actions/EditNote.actions.ts

```diff
@@ -46,12 +46,12 @@ export const editNoteRequest = note => dispatch => {
     // })
 
     return axios.request({
-        url:`${baseName}/note/${note.id}/`,
+        url:`${baseName}/noe/${note.id}/`,
         method: 'PUT',
         data: note
     })
     .then(response => dispatch(handleEditNoteSuccess(response.data)))
-    .catch(error => dispatch(handleEditNoteFail(error)));
+    .catch(error => dispatch(handleEditNoteFail('error, ee')));
```

Comment:
Hardcoded stuff? You can just dispatch an action with no payload (only type) and hardcode the value in reducer.